manipvortex/manipulator.go

```diff
@@ -631,7 +636,7 @@ func (m *vortexManipulator) backgroundSync(ctx context.Context) {
 
 			// Update the local copy of the object now.
 			if err := m.commitLocal(t.Method, t.mctx, t.Object); err != nil {
-				zap.L().Error("failed to delete local object after failed resync", zap.Error(err))
+				zap.L().Error("failed to commit object downstream", zap.Error(err))
```

Comment:
Commit local should be controlled with an option. I believe we have this in other places of the code, but in the case where it is just a buffer, we don't always need a local copy. This will enhance the API.

Reply:
Yes. I discussed with @primalmotion about this and will work on it in a separate branch.